code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -1480,11 +1481,24 @@
 		var/drowsiness_to_apply = max(round(reac_volume, 1) * 2 SECONDS, 4 SECONDS)
 		exposed_mob.adjust_drowsiness(drowsiness_to_apply)
 
+/datum/reagent/nitrous_oxide/on_mob_metabolize(mob/living/affected_mob)
+	if(!HAS_TRAIT(affected_mob, TRAIT_COAGULATING)) //IF the mob does not have a coagulant in them, we add the blood mess trait to make the bleed quicker
+		ADD_TRAIT(affected_mob, TRAIT_BLOODY_MESS, /datum/reagent/nitrous_oxide)
```

Comment:
```suggestion
		ADD_TRAIT(affected_mob, TRAIT_BLOODY_MESS, type)
```

same result with less typing, and a bit more polite in case some deranged person decides to subtype this (they shouldn't though that's a bit weird). tres bon